### DISCUSSION TITLE
How to change default keybinds

### DISCUSSION THREAD

**Discussion by freddieventura** [browsh-org/browsh#498] [General]
HI there,

First off big thanks to the Devs (or mostly the Dev) for such a great idea and materialization of it. I works quite nicely, and graphics are awesome.
My workflow is CLI based so I have actually been looking for this tool quite some time. The issue is that my keybinds are conflicting.
So Ctrl + l is currently taken by tmux , how can I change the default keybinds?

Thank you

**Comment by tombh** [comment]
Thanks for the compliment!

I'm afraid custom keybindings aren't possible at the moment 😞 There's an existing issue you can subscribe to https://github.com/browsh-org/browsh/issues/146
